modules/config/i18n.nix

```diff
@@ -36,6 +37,27 @@ let
 in {
   meta.maintainers = with maintainers; [ midchildan ];
 
+  options = {
+    i18n.glibcLocales = mkOption {
+      type = types.path;
+      visible = false;
+      internal = true;
```

Comment:
Why not make this visible?

Reply:
OK @ncfavier - I have made the option visible and enhanced the doc texts.